docker/hive/health-check.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ev
+
+HIVE_SERVER_PORT=${HIVE_SERVER_PORT:=10000}
+
+echo Health check for Hive:
+/opt/hive/bin/beeline -u "jdbc:hive2://127.0.0.1:${HIVE_SERVER_PORT}/default;auth=noSasl" -n root -p pass -e "show tables;"
```

Comment:
If Hive is unresponsive, so an error will be raised. Are we gonna see it in logs?

Reply:
Allow setting the user/password from outside?

Reply:
These are mock creds since we use noSasl

Reply:
@lyogev Apparently NoSasl requires beeline user but not password. I changed the user to `health_check`